tests/testthat/helper.R

```diff
@@ -6,15 +6,15 @@ expect_tuner = function(tuner) {
     public = c("optimize", "param_set"),
     private = ".optimize"
   )
-  expect_is(tuner$param_set, "ParamSet")
+  expect_s3_class(tuner$param_set, "ParamSet")
```

Comment:
Can't we leave it as it is? Because it is not really an S3 class - only technically.

Reply:
The old form is deprecated and will error at some point.

`expect_class` might also work.